flow360/component/flow360_params/solvers.py

```diff
@@ -229,13 +229,15 @@ class IncompressibleNavierStokesSolver(GenericFlowSolverSettings):
 class TurbulenceModelConstantsSA(Flow360BaseModel):
     """:class:`TurbulenceModelConstantsSA` class"""
 
+    model_type: Literal["SA"] = pd.Field("SA", alias="modelType", const=True)
     C_DES: Optional[float] = pd.Field(0.72)
     C_d: Optional[float] = pd.Field(8.0)
 
 
 class TurbulenceModelConstantsSST(Flow360BaseModel):
     """:class:`TurbulenceModelConstantsSST` class"""
 
+    model_type: Literal["SST"] = pd.Field("SST", alias="modelType", const=True)
```

Comment:
same here, lest use full name, kOmegaSSTConsts

Reply:
Fixed, renamed to kOmegaSSTConsts